ui/raidboss/data/07-dt/raid/r6s.ts

```diff
@@ -10,10 +11,22 @@ import { RaidbossData } from '../../../../../types/data';
 import { NetMatches } from '../../../../../types/net_matches';
 import { TriggerSet } from '../../../../../types/trigger';
 
+// TODOs:
+// - clean up timeline
+// - Crowd Brûlée - party stack (non-defamations)?
+// - safe corners for quicksand?
+// - Live Painting - add wave # spawn call?
+// - Ore-rigato - Mu enrage
+// - Hangry Hiss - Gimme Cat enrage
+// - Mousse Drip - ranged baits into 4x pair stacks/puddle drops
+// - Moussacre - melee bait proteans
+// - tower soaks
```

Comment:
Maybe also add a TODO here about tracking the thunderstorm movement so that we can give the player a "rotate clockwise"/"rotate counterclockwise" trigger?